<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Hallinnointiohjelma</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="programsTab">
       <attribute name="title">
        <string>Koulutusohjelmat</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="programsWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="programsCountLabel">
             <property name="text">
              <string>Koulutusohjelmien määrä: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="programsListView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="programsButtonsWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="addProgramButton">
                <property name="text">
                 <string>Lisää</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteProgramButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Poista</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="detailsWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="detailsLabel">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Ohjelma</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="detailsGroupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QGroupBox" name="programTeachersGroupBox">
                <property name="title">
                 <string>Opettajat</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QListView" name="programsTeachersListView"/>
                 </item>
                 <item>
                  <widget class="QWidget" name="programTeachersButtonsWidget" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QPushButton" name="addTeacherToProgramButton">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Lisää</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="removeTeacherFromProgramButton">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Poista</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="programStudentsGroupBox">
                <property name="title">
                 <string>Oppilaat</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QListView" name="programStudentsListView"/>
                 </item>
                 <item>
                  <widget class="QWidget" name="programStudentsButtonsWidget" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QPushButton" name="addStudentToProgramButton">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Lisää</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="removeStudentFromProgramButton">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Poista</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="teachersTab">
       <attribute name="title">
        <string>Opettajat</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QWidget" name="teachersWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="teacherCountLabel">
             <property name="text">
              <string>Opettajien määrä:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="teachersListView"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="addTeacherButton">
               <property name="text">
                <string>Lisää opettaja</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteTeacherButton">
               <property name="text">
                <string>Poista opettaja</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="teacherProgramsWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="teacherProgramsLabel">
             <property name="text">
              <string>Opettajan koulutusohjelmat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="teacherProgramsListView"/>
           </item>
           <item>
            <widget class="QPushButton" name="removeTeacherFromProgramButton_2">
             <property name="text">
              <string>Poista Koulutusohjelmasta</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addTeacherToProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddTeacherToProgramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeTeacherFromProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveTeacherFromProgramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addStudentToProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddStudentToProgramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeStudentFromProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveStudentFromProgramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddProgramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteProgramButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveProgramClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTeacherButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAddTeacherClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteTeacherButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveTeacherClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeTeacherFromProgramButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRemoveTeacherFromProgramClicked2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAddTeacherToProgramClicked()</slot>
  <slot>onRemoveTeacherFromProgramClicked()</slot>
  <slot>onAddStudentToProgramClicked()</slot>
  <slot>onRemoveStudentFromProgramClicked()</slot>
  <slot>onAddProgramClicked()</slot>
  <slot>onRemoveProgramClicked()</slot>
  <slot>onAddTeacherClicked()</slot>
  <slot>onRemoveTeacherClicked()</slot>
  <slot>onRemoveTeacherFromProgramClicked2()</slot>
 </slots>
</ui>
